docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -27,25 +27,23 @@ The main features of the Sensor API include:
 
 - Sensor types
 
-  A device can have various physical and virtual sensors. The following table lists the sensors supported by Tizen.
+  A device can have various physical and virtual sensors. The following table lists the sensors supported by Tizen:
 
   > **Note**
   >
   > Not all devices support all sensors, so each sensor is not necessarily available on all devices. You can [check whether a sensor is supported](#sensorlistener). For more information, see [System Information](../device/system.md).
 
-  **Table: Supported sensor types**
-
-  | Supported sensor types                   |                                          |                                          |
-  |------------------------------------------|------------------------------------------|------------------------------------------|
-  | [Accelerometer](#accelerometer)          | [Heart Rate Monitor Sensor](#hrm)        | [Proximity Sensor](#proximity)           |
-  | [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Humidity Sensor](#humidity)             | [Rotation Vector Sensor](#rotation)      |
-  | [Gravity Sensor](#gravity)               | [Light Sensor](#light)                   | [Significant Motion Sensor](#significant_motion) |
-  | [Gyroscope](#gyro)                       | [Linear Acceleration Sensor](#lin_accelerometer) | [Sleep Monitor](#sleep_monitor)          |
-  | [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Magnetic Sensor](#magnetic)             | [Temperature Sensor](#temperature)       |
-  | [Heart Rate Monitor LED Green Sensor](#hrm_green) | [Orientation Sensor](#orientation)       | [Ultraviolet Sensor](#ultraviolet)       |
-  | [Heart Rate Monitor LED IR Sensor](#hrm_ir) | [Pedometer](#pedometer)                  | [Uncalibrated Gyroscope](#uncal_gyro)    |
-  | [Heart Rate Monitor LED Red Sensor](#hrm_red) | [Pressure Sensor](#pressure)             | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |
-
+  |                                                               |                                                  |                                                  |
+  |---------------------------------------------------------------|--------------------------------------------------|--------------------------------------------------|
```

Comment:
@bbrto21, @nawab-samsung 
I am sorry looks like now both the Table caption and header row title both are removed now. I think one of them should be retained. See https://github.com/Samsung/tizen-docs/pull/1383/commits/19a089eeda37bbd8138c64070ddb002a80db93a5#r550416548

I recommend to add back the header row 
| Supported sensor types                                        |                                                  |                                                  |

Reply:
ok, I`ll fix it